docs/migrating_to_5.md

```diff
@@ -1,5 +1,8 @@
 # Migrating from 4.x to 5.x
 
+Please note: we plan to discontinue Mongoose 5 support on March 1, 2024.
```

Comment:
I'm not sure if I'm not understanding correctly but from reading this it appears mongoose 5 is going to be supported longer than mongoose 6?

Reply:
Nope, we're discontinuing support for Mongoose 5 on March 1, 2024. We don't have any plan to discontinue support for Mongoose 6 currently.

Reply:
probably because of
docs/version-support.md
> We will continue to ship bug fixes to Mongoose 6 until August 24, 2023.